tests/utils.test.js

```diff
@@ -134,6 +135,20 @@ describe("Test utils.js", () => {
       borderColor: "#fff",
     });
   });
+
+  it("formatBytes: should return expected values", () => {
+    expect(formatBytes(0)).toBe("0.0 B");
```

Comment:
I have some doubts about that bytes have decimal part. I think it should be just 0 B.

Reply:
Okay. I removed unnecessary decimal parts at 7e17b50